templates/tomcat/cert-users.properties.erb

```diff
@@ -1 +1 @@
-katelloUser=<%= scope['::candlepin::artemis_client_dn'] %>
+katelloUser=<%= @artemis_client_dn %>
```

Comment:
Given the simplicity, I wonder if it really needs to be a template. An inline string could be good enough.

Reply:
Looks like this file can now be removed